byron/ledger/executable-spec/test/Ledger/Delegation/Properties.hs

```diff
@@ -245,27 +277,38 @@ expectedDms s d sbs = Bimap.fromList (fmap (delegator &&& delegate) activeCerts)
     activationSlot :: Int
     activationSlot = s - d
 
-    -- | Get the active certificates from each block, and concatenate them all
-    -- together.
-    activeCerts :: [DCert]
-    activeCerts = concatMap _blockCerts activeBlocks
+delegatorDelegate :: DCert -> (VKeyGenesis, VKey)
+delegatorDelegate = delegator &&& delegate
 
 instance HasTrace DBLOCK where
 
-  initEnvGen = do
-    n <- integral (linear 0 14)
-    e <- integral (linear 0 1000)
-    s <- integral (linear 0 1000)
-    k <- integral (linear 1 10000)
-    pure $! DSEnv (Set.fromAscList $ gk <$> [0..n]) (Epoch e) (Slot s) (BlockCount k)
+  initEnvGen
+    = DSEnv
+    <$> allowedDelegators
+    -- We do not expect the current epoch to have an influence on the tests, so
+    -- we chose a small value here.
+    <*> epochGen 0 10
+    -- As with epochs, the current slot should not have influence in the tests.
+    <*> slotGen 0 100
```

Comment:
is  `100` then a good value here if "small" is desired?

Reply:
Yeah, good point.